frontend/src/__tests__/unit/resourceEquality.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { isCpuLimitEqual, isMemoryLimitEqual } from '~/utilities/valueUnits';
+
+test('Cpu limit - correctly compares non-undefined values', () => {
+  expect(isCpuLimitEqual('1', '1')).toBe(true);
+  expect(isCpuLimitEqual('1000m', '1')).toBe(true);
+  expect(isCpuLimitEqual('1001m', '1')).toBe(false);
+});
+
+test('Cpu limit - correctly compares undefined values', () => {
+  expect(isCpuLimitEqual('1000m', undefined)).toBe(false);
+  expect(isCpuLimitEqual('1', undefined)).toBe(false);
+  expect(isCpuLimitEqual(undefined, undefined)).toBe(true);
+});
+
+test('Memory limit - correctly compares non-undefined values', () => {
+  expect(isMemoryLimitEqual('1Gi', '1Gi')).toBe(true);
+  expect(isMemoryLimitEqual('1Gi', '1024Mi')).toBe(true);
+  expect(isMemoryLimitEqual('1Gi', '1025Mi')).toBe(false);
+});
+
+test('Memory limit - correctly compares undefined values', () => {
+  expect(isMemoryLimitEqual('1Gi', undefined)).toBe(false);
+  expect(isMemoryLimitEqual('1024Mi', undefined)).toBe(false);
+  expect(isMemoryLimitEqual(undefined, undefined)).toBe(true);
+});
```

Comment:
```suggestion
describe('isCpuLimitEqual', () => {
  test('correctly compares non-undefined values', () => {
    expect(isCpuLimitEqual('1', '1')).toBe(true);
    expect(isCpuLimitEqual('1000m', '1')).toBe(true);
    expect(isCpuLimitEqual('1001m', '1')).toBe(false);
  });

  test('correctly compares undefined values', () => {
    expect(isCpuLimitEqual('1000m', undefined)).toBe(false);
    expect(isCpuLimitEqual('1', undefined)).toBe(false);
    expect(isCpuLimitEqual(undefined, undefined)).toBe(true);
  });
});

describe('isMemoryLimitEqual', () => {
  test('correctly compares non-undefined values', () => {
    expect(isMemoryLimitEqual('1Gi', '1Gi')).toBe(true);
    expect(isMemoryLimitEqual('1Gi', '1024Mi')).toBe(true);
    expect(isMemoryLimitEqual('1Gi', '1025Mi')).toBe(false);
  });

  test('correctly compares undefined values', () => {
    expect(isMemoryLimitEqual('1Gi', undefined)).toBe(false);
    expect(isMemoryLimitEqual('1024Mi', undefined)).toBe(false);
    expect(isMemoryLimitEqual(undefined, undefined)).toBe(true);
  });
});
```

Reply:
@tkolo Could you make the suggested change to group the tests.

Reply:
Ops, forgot about that part, sorry about that. Fixed